docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -86,8 +86,9 @@ Elements related to domain [startup and shutdown]({{< relref "/userguide/managin
 
 Elements related to overriding WebLogic domain configuration:
```

Comment:
The new introspectorJobActiveDeadlineSecodns is missing under configuration.

Reply:
I'll add it.

Reply:
@doxiao I just added it. Please take a look and resolve this if correct.